### PR TITLE
Fe 534 FE-867 Prepare for token launch

### PR DESCRIPTION
**Pull request by ipanagiv**

### CODE REVIEW

File: docs/wallet/add-edit-wallet-address.mdx [hunk 1]
```diff
@@ -74,7 +74,7 @@ To edit or change your wallet address, go to settings screen and tap "My wallet"
 ## How to create wallet on Metamask
 
 While we are on the testnet, we suggest you use a **MetaMask** wallet with WeatherXM. By following our guide, it's
```
Comment (PavlosTze) [inline docs/wallet/add-edit-wallet-address.mdx:76]: This is wrong. Replace this. We need to write something that will say that we have tested Metamask etc. and in the future we will add a paragraph showing the supported wallets.

File: docs/wallet/add-edit-wallet-address.mdx [hunk 2]
```diff
@@ -103,5 +103,5 @@ Always check if you are using the official website to make sure you’re install
 1. Your MetaMask wallet will now be set up and ready to use. Click [All Done] to view your wallet.
 
 :::note
-If you want to confirm the existence of your tokens in your preferred wallet, you need to add our ERC20 token's smart contract.
+If you want to confirm the existence of your tokens in your preferred wallet, you need to add our ERC20 token's smart contract.See how [here](https://docs.weatherxm.com/wallet/how-to-join-arbitrum-and-see-your-tokens#add-wxm-token-in-metamask)
```
Comment (PavlosTze) [inline docs/wallet/add-edit-wallet-address.mdx:106]: Leave a space at the dot at "smart contract.See how "

File: docs/the-$WXM-token.mdx [hunk 3]
```diff
@@ -23,12 +23,12 @@ $WXM will be used to govern the Network and will be required to license weather
  **$WXM** is a an ERC20 token to power the weather data economy.
 
 :::note
-WeatherXM is based on an EVM compatible chain - we are currently developing on Arbitrum Sepolia Testnet.
+WeatherXM is deployed at Arbitrum One Network. You can find the contract at [Arbitrum One Explorer](https://arbiscan.io/token/0xb6093b61544572ab42a0e43af08abafd41bf25a6)
```
Comment (PavlosTze) [inline docs/the-$WXM-token.mdx:26]: deployed at ---> deployed on

File: docs/rewards/rewards-claiming.mdx [hunk 4]
```diff
@@ -69,7 +69,12 @@ To claim your rewards:
 </Tabs>
 
 
+### Bridging Your Assets from Ethereum to Arbitrum:
+ 1. Go to https://bridge.arbitrum.io/
+ 2. Then, simply enter the amount of ETH you wish to use on Arbitrum and click “Deposit”. 
+ 3. Next, you will need to confirm the transaction in your wallet. Please note that this transaction is on Layer 1 and standard gas fees will apply. 
+ However, once on Arbitrum you can enjoy lower gas fees. Once the amount is correct, click “Confirm”
```
Comment (PavlosTze) [inline docs/rewards/rewards-claiming.mdx:76]: Please add a dot at the end.

File: docs/rewards/rewards-claiming.mdx [hunk 5]
```diff
@@ -69,7 +69,12 @@ To claim your rewards:
...
+ 4. Once confirmed, you will see the transaction appear and process at the bottom of your screen. This can take a few minutes.
+ 5. If you require additional support to bridge your assets, you can refer to Arbitrum’s tutorial [here](https://docs.arbitrum.io/getting-started-users).
 
-<hr/>
-
-
+ **This Guide is only on desktop with the MetaMask browser extension already installed.**
```
Comment (PavlosTze) [inline docs/rewards/rewards-claiming.mdx:80]: This guide is only on ----> This guide is applicable only for

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Good job @ipanagiv , some minor changes needed which are mentioned below. Also, consider changing these ones also:
1. https://docs.weatherxm.com/faqs#can-i-change-my-wallet-address-in-the-future (testnet mention)
2. https://docs.weatherxm.com/project/rewards-troubleshooting (the two notes regarding the "PoL penalization while on testnet").

File: docs/wallet/add-edit-wallet-address.mdx [hunk 6]
```diff
@@ -73,8 +73,8 @@ To edit or change your wallet address, go to settings screen and tap "My wallet"
 
 ## How to create wallet on Metamask
 
-While we are on the testnet, we suggest you use a **MetaMask** wallet with WeatherXM. By following our guide, it's
-fairly simple to set up a MetaMask wallet and add a custom token (in our case, the WXM token) on Arbitrum One Network.
+#WXM is deployed on Arbitrum One Network. We suggest you use a **MetaMask** wallet with WeatherXM, as we have extensivelly tested on it. As we evolve, we will official support several wallets.
```
Comment (PavlosTze) [inline docs/wallet/add-edit-wallet-address.mdx:76]: #WXM -> $WXM

**Review by PavlosTze — CHANGES_REQUESTED** [review]
1. https://docs.weatherxm.com/faqs#can-i-change-my-wallet-address-in-the-future (testnet mention)
2. https://docs.weatherxm.com/project/rewards-troubleshooting (the two notes regarding the "PoL penalization while on testnet").

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Please handle this @ipanagiv https://docs.weatherxm.com/faqs#can-i-change-my-wallet-address-in-the-future (testnet mention)

**Review by PavlosTze — APPROVED** [review]

**Review by PavlosTze — APPROVED** [review]

**Review by PavlosTze — CHANGES_REQUESTED** [review]
@ipanagiv Remove the images you do not use anymore, as seen here: https://github.com/WeatherXM/docs/pull/84/commits/3436b4cc9daae7144e7d8bc7c5b0b7de50f6e18d

**Review by NikosTsiligaridis — APPROVED** [review]

**Pull request merged**